src/shell/shell_cmds.cpp

```diff
@@ -132,11 +132,12 @@ bool lookup_shell_cmd(std::string name, SHELL_Cmd &shell_cmd)
 	return true;
 }
 
-bool DOS_Shell::ExecuteConfigChange(const char* const cmd_in, const char* const line)
+bool DOS_Shell::ExecuteConfigChange(const char* const cmd_in, const std::string args)
```

Comment:
Minor but you could take in `std::string_view` to avoid a copy. It's the more direct C++ equivalent to `char *`.

Reply:
Yeah I'll try that.

Reply:
Actually, I can't because I'm calling `c_str()` on it later. The big showstopper with calling `data()` on `std::string_view` is that the terminating zero is not guaranteed (which kinda makes sense, but it's bad for C interop as you need to make a copy anyway to be 100% safe).